article-store/events/conf.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+import pika
+
+
+RABBITMQ_URL = os.environ.get('RABBITMQ_URL')
+
+BROKER_PARAMS = pika.connection.URLParameters(RABBITMQ_URL)
+
+ARTICLES_QUEUE = os.environ.get('ARTICLES_QUEUE', 'articles')
```

Comment:
I agree with other comments that a queue should not be necessary here. If we want to set up one for testing, it should be clearly named as `articles-dummy` or similar